mirrord/config/src/internal_proxy.rs

```diff
@@ -84,6 +84,22 @@ pub struct InternalProxyConfig {
     #[config(default = 5)]
     pub idle_timeout: u64,
 
+    /// ### internal_proxy.breakpoint_timeout {#internal_proxy-breakpoint_timeout}
+    ///
+    /// Sometimes the cpu is too busy with other tasks and the internal proxy is blocked
+    /// from performing its keep-alive check. Increase this value if mirrord exits while
+    /// you're debugging with breakpoints.
+    ///
+    /// ```json
+    /// {
+    ///   "internal_proxy": {
+    ///     "idle_timeout": 31536000
+    ///   }
+    /// }
```

Comment:
This is kind of misleading :v
1. The default value is already at max, so "Increase this value if mirrord exits while you're debugging with breakpoints" does not sound right
2. Snippet contains `idle_timeout` field
3. The explanation itself is kinda off, the problem is not about keep-alive checks

This setting is not really for internal proxy but for the layer, since we set the timeout there. I propose we move this into `experimental` section under `.experimental.intproxy_socket_timeout` without any instructions (just a doc saying that it's a read/write timeout for layer<->intproxy sockets, the users should not normally touch it without our directions)

Reply:
I don't think the user should be seeing this either, but it feels like it belong under the `internal_proxy` config. I mean, this is a setting for an internal proxy startup thingy. Aren't the `experimental` configs for things we're not quite sure we want? This one feels like it doesn't hurt to have, so I see no reason to remove it in the future.

Reply:
Tbh I think `experimental` is mainly for hacks :V

> this is a setting for an internal proxy startup thingy

Wait, no. You're thinking about `.internal_proxy.idle_timeout`. My comment is for your new setting. The timeout specified in it is set on layer's TCP socket for communicating with intproxy (so applies to any request<->response exchange)

Reply:
> layer's TCP socket for communicating with intproxy

Yeah I got that, but isn't that part of the proxy startup? When we have a new layer connection? That's what I mean.

Reply:
Anyway, I changed the docs.

Reply:
Oke, let's leave it in `.internal_proxy` section